java-lib/src/main/java/com/wavefront/ingester/AbstractIngesterFormatter.java

```diff
@@ -38,10 +43,16 @@
 
   protected final List<FormatterElement<T>> elements;
 
+  protected static IngesterContext ingesterContext = new IngesterContext.Builder().build();
```

Comment:
You shouldn't use a static field? This makes the context global for all. Do create a DEFAULT_INGESTER_CONTEXT and bind the local default to it